src/CodeFormatter/App.config

```diff
@@ -1,42 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
-    </startup>
-  <runtime>
-    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
Please add this file back.

These bindings are needed on machines which have older versions of MSBuild / Roslyn installed into the GAC.